makoctl

```diff
@@ -33,22 +37,39 @@ fi
 
 case "$1" in
 "dismiss")
-	[ $# -lt 2 ] && action="" || action="$2"
-	case "$action" in
-	"-a"|"--all")
+	all=0
+	group=0
+	id=0
+	while [ $# -gt 1 ]; do
```

Comment:
Why do we need a loop here? Wasn't the previous code enough?

Reply:
So that you can use `-g` with `-n <id>`

Reply:
Ah, makes sense